src/timeline/js/directives/clip.js

```diff
@@ -211,21 +211,6 @@ App.directive("tlClip", function ($timeout) {
           }
         }
       });
-
-      //handle hover over on the clip
-      element.hover(
-        function () {
-          if (!dragging) {
-            element.addClass("highlight_clip", 200, "easeInOutCubic");
-          }
-        },
-        function () {
-          if (!dragging) {
-            element.removeClass("highlight_clip", 200, "easeInOutCubic");
-          }
-        }
-      );
-
```

Comment:
I knew I really wanted to get rid of this, because it was fighting with the rest of the CSS rules. You could even see it happening in the UI, clips would get into a state where mousing between them would cause borders to flash, or disappear momentarily and reappear — all sorts of issues. Plus, `:hover` styling is at this point a basic CSS feature, there's no reason it couldn't be used to create the hover effects without needing to add any extra JavaScript code.